operations/update.go

```diff
@@ -25,7 +25,7 @@ import (
 func Update() cli.Command {
 	const installFlagName = "install"
 	const forceFlagName = "force"
-	const autoUpdateFlagName = "auto_update"
+	const autoUpgradeFlagName = "auto_update"
```

Comment:
I would have this value be just "auto" (or at least "auto-upgrade" for consistency with wording and dashes)